app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -48,7 +48,7 @@ import java.util.*
  * The V in MVP
  * Created by chiragw15 on 9/7/17.
  */
-class ChatActivity: AppCompatActivity(), IChatView {
+class ChatActivity : AppCompatActivity(), IChatView {
```

Comment:
No space should be there before colon

Reply:
OK. But refrain from formatting code in PRs, use separate PR for such stuff

Reply:
Sure @iamareebjamal removing this space